cmd/ftl/cmd_serve.go

```diff
@@ -293,6 +293,11 @@ func (s *serveCommonConfig) run(
 				Types:       []schema.ResourceType{schema.ResourceTypeRunner},
 				ID:          "runner",
 			},
+			{
+				Provisioner: provisioner.NewFixtureProvisioner(),
```

Comment:
How do we express that this needs to run after everything else?

Reply:
AFAIK it is the order of of the provisioners list here.